pkg/tsdb/cloudwatch/models/types.go

```diff
@@ -6,13 +6,14 @@ import (
 	"github.com/grafana/grafana-plugin-sdk-go/backend"
 )
 
-type Clients struct {
+type RequestContext struct {
 	MetricsClientProvider MetricsClientProvider
+	Settings              *CloudWatchSettings
```

Comment:
The namespace route needs access to the ds instance, so rather than resolving just the clients we now resolve a full request context struct